.github/workflows/tests.yml

```diff
@@ -41,3 +41,113 @@ jobs:
 
             -   name: Run tests
                 run: ./vendor/bin/simple-phpunit
+
+    test-windows:
+        name: PHPUnit Tests (Windows 7.1 x86)
+        runs-on: windows-latest
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-main
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Cache binaries
+                id: cache-binaries
+                uses: actions/cache@v2
+                with:
+                    path: c:\php
+                    key: ${{ runner.os }}-binaries-${{ hashFiles('**/.github/workflows/tests.yml') }}
+                    restore-keys: ${{ runner.os }}-binaries-
+
+            -   name: Install binaries
+                if: steps.cache-binaries.outputs.cache-hit != 'true'
+                run: |
+                    echo ::group::Download-Php
+                    if(!(Test-Path -path "c:\php\"))
+                    {
+                      New-Item -ItemType directory -Path "c:\php"
+                    }
+                    cd c:\php
+                    curl.exe -fsL https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.1.3-Win32-VC14-x86.zip -o php.zip
+                    7z x php.zip -y
+                    del ./*.zip
+                    echo ::endgroup::
+
+                    echo ::group::Download-extensions
+                    cd ext
+                    curl.exe -fsL https://github.com/symfony/binary-utils/releases/download/v0.1/php_apcu-5.1.8-7.1-ts-vc14-x86.zip -o php_apcu.zip
+                    7z x php_apcu.zip -y
+                    del ./*.zip
+                    cd ..
+                    echo ::endgroup::
+
+                    echo ::group::Setup-.ini
+                    echo @php %%~dp0composer.phar %%* > composer.bat
+                    copy php.ini-development php.ini
+                    echo max_execution_time=1200 >> php.ini
+                    echo date.timezone="UTC" >> php.ini
+                    echo memory_limit=-1 >> php.ini
+                    echo extension_dir=ext >> php.ini
+                    echo extension=php_openssl.dll >> php.ini
+                    echo extension=php_apcu.dll >> php.ini
+                    echo extension=php_intl.dll >> php.ini
+                    echo extension=php_mbstring.dll >> php.ini
+                    echo extension=php_ldap.dll >> php.ini
+                    echo ::endgroup::
+
+                    echo ::group::Download-composer
+                    curl.exe -fsL https://github.com/composer/composer/releases/download/2.0.0/composer.phar -o composer.phar
```

Comment:
why hardcoding 2.0.0 by taking it from github release instead of taking `https://getcomposer.org/composer-stable.phar` which gets the latest stable release ? 2.0.0 is already outdated by 2 releases.

Reply:
thanks